inventory-schemas/system_profile_schema.yaml

```diff
@@ -169,7 +169,6 @@ $defs:
         description: The origin repo from which the commit was installed
         example: "fedora/33/x86_64/silverblue, fedora/31/x86_64/blackpink, fedora/34/x86_64/orangeblue"
         type: string
-        minLength: 1
```

Comment:
was this line causing any problems? Is it possible to a empty origin repo?

Reply:
The schema is synced from the inventory-schemas repo. This line was removed by https://github.com/RedHatInsights/inventory-schemas/pull/127